### ISSUE TITLE
batch delete

### ISSUE THREAD

**Issue by pryme0** [supabase/postgrest-js#292]
### Discussed in https://github.com/supabase/supabase/discussions/4831

<div type='discussions-op-text'>

<sup>Originally posted by **pryme0** January  6, 2022</sup>
Is there a way to do batch delete with supbase?
Like i pass in an array containing id's of records i want to delete  and it runs the operation for me</div>

**Comment by steve-chavez** [comment]
@aiibe Yes, see https://github.com/PostgREST/postgrest/issues/2314. A bulk delete with a request payload is being worked on https://github.com/PostgREST/postgrest/pull/2355.

**Comment by soedirgo** [comment]
Repurposing this issue to batch delete w/ request body
